### PR TITLE
Module types was renamed to module linking

### PR DESCRIPTION
**Pull request by littledan**
Is it still Phase 0 or did it go to Phase 1 at some point?

### CODE REVIEW

**Comment by binji** [comment]
Yes, it should be phase 1. (In fact, I don't think any of the proposals listed here should be phase 0)

**Comment by chicoxyzzy** [comment]
I just made a quick search in the meetings repo and it seems that only Module linking was promoted to phase 1 and all other proposals in the phase 0 list are still phase 0 yet.

**Comment by chicoxyzzy** [comment]
Module linking  was promoted to phase 2 in [June](https://github.com/WebAssembly/meetings/blob/642490e97b99787d13f752b92801c874a61e5c9d/main/2020/CG-06-09.md#update-on-module-types--module-linking-proposal-slides)

**Comment by binji** [comment]
OK, let's move this up to phase 1 then, along with the rename.

**Comment by lukewagner** [comment]
+1 thanks!

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Proposals follow [this process document](https://github.com/WebAssembly/meetings
 | ---------------------------------------------------------- | -------------------------------- |
 | [Web Content Security Policy][web_content_security_policy] | Ben Titzer                       |
 | [Funclets: Flexible Intraprocedural Control Flow][funclets]| Dan Gohman                       |
-| [Module Types][module_types]                               | Luke Wagner and Andreas Rossberg |
+| [Module Linking][module_linking]                           | Luke Wagner and Andreas Rossberg |
```
Comment (chicoxyzzy) [inline README.md:59]: please move this proposal to phase 1 proposals list

**Comment by chicoxyzzy** [comment]
@littledan thanks! I've rebased on master and moved it to phase 1

**Pull request merged**